run.sh

```diff
@@ -31,9 +31,13 @@ if ! hash jq 2>/dev/null; then
   echo 2>&1 "No 'jq' available."
   exit 1
 fi
-if ! hash speed-test 2>/dev/null; then
```

Comment:
You still need this check, the npm tools directory might not be in the PATH.
In this scenario this check fails and `npm list -g` succeeds. In that case, you could manually add it to the PATH before running.

Reply:
Got it,
What do you think about adding the check for npm executable and stop script execution if it not found?
```sh
if ! hash npm 2>/dev/null; then
exit 1
fi 
```

Reply:
It is a good addition too.